userservice/src/main/resources/application.properties

```diff
@@ -5,10 +5,6 @@ server.port=8080
 spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
 springfox.documentation.swagger.v2.path=/api/v2/api-docs
 
-# Redis configuration
-spring.data.redis.host=localhost
-spring.data.redis.port=6379
```

Comment:
Why did you remove the Redis configuration?

Reply:
**FYI**, @huyluong22 , @thaibinh156 
Please keep the original value for configuration. If you want to customize your configuration to be compact with your local development machine, you can use environment variables set up on IntellJ.
For example:
![image](https://github.com/user-attachments/assets/ecf13b76-3801-41c0-830d-a93eaeacfb0d)